pkg/kubelet/prober/prober.go

```diff
@@ -202,14 +202,20 @@ func (pb *prober) runProbe(ctx context.Context, probeType probeType, p *v1.Probe
 type execInContainer struct {
 	// run executes a command in a container. Combined stdout and stderr output is always returned. An
 	// error is returned if one occurred.
-	run    func() ([]byte, error)
-	writer io.Writer
+	run           func() ([]byte, error)
+	writer        io.Writer
+	pod           *v1.Pod
+	containerName string
```

Comment:
nit: Storing pod and container would be more consistent.

Reply:
Done